tests/lib/utils/context.php

```diff
@@ -47,6 +47,24 @@ public function test__is_maintenance_mode__yep() {
 		$this->assertTrue( $actual_result );
 	}
 
+	public function test__is_overdue_locked__nope() {
+		// Note: `VIP_OVERDUE_LOCKOUT` not defined
+
+		self::assertFalse( defined( 'VIP_OVERDUE_LOCKOUT' ) );
+
+		$actual_result = Context::is_overdue_locked();
+
+		$this->assertFalse( $actual_result, '`VIP_OVERDUE_LOCKOUT` constant should not be defined for this test' );
```

Comment:
```suggestion
		$this->assertFalse( $actual_result, 'The `VIP_OVERDUE_LOCKOUT` constant is not defined, but site is reported as locked' );
```

The text of the error message doesn't reflect what's being asserted.